### ISSUE TITLE
Scrolling is not smooth on Android

### ISSUE THREAD

**Issue by parkkralle** [Clooos/Bubble-Card#253]
Hello,

I have performance problems with the popup-card. On a Pixel 5 with Android 14 and a Pixel 3 with Android 12, the popup card is very jerky when scrolling. The animation when opening the popup-card is also jerky. The popup card jerks in the companion app as well as in the chrome browser. An old version (1.0.3) runs absolutely smoothly. On my iOS devices I have no stuttering at all with the latest version.

**Comment by Clooos** [comment]
Hi and thanks for your feedback.

This issue should be fixed in the new 1.6.0-beta.2 🤞
https://github.com/Clooos/Bubble-Card/releases/tag/v1.6.0-beta.2

**Comment by parkkralle** [comment]
Unfortunately the problem still exists in 1.6.0-beta.2

**Comment by Clooos** [comment]
You are not the first to stay on 1.0.3, I will take a closer look at that!

**Comment by Clooos** [comment]
Have you tried to add `bg_blur: 0`, `bg_opacity: 0` and `shadow_opacity: 0` to your pop-ups? It seems that this is the major difference between 1.0.3 and 1.1.0 that could cause your issue.

**Comment by parkkralle** [comment]
It's not as smooth as version 1.0.3 but significantly better. I played around a bit with the three parameters and think it's because of the blur option. The other two options have shown no effect on scrolling. I suspect that with blur 0 the blur function is still active and has a negative effect on scrolling. I remember that I also had problems with a theme that used blur.
If you tell me which lines of code I should delete so that blur is completely deactivated, I would test it again.

**Comment by Clooos** [comment]
You can fully disable it like that in YAML:

```yaml
type: vertical-stack
cards:
  - type: custom:bubble-pop-up
    hash: '#your-pop-up'
    styles: | 
      #root {
        backdrop-filter: none !important;
      }
```

> PS: How can I use both versions in parallel to compare them better?

I have some ideas but it's way too complicated 😅

**Comment by parkkralle** [comment]
With this option it is just as smooth as in version 1.0.3. So it actually seems to be due to the blur effect. Do you have any idea why?

**Comment by Clooos** [comment]
I really don't understand how you can have this issue on two high-end Android devices, there is maybe an issue with your devices because on my Android 14 phone it's really smooth (full 60fps).

**Comment by parkkralle** [comment]
I'll ask my friend with a Pixel 8 Pro if he can test the PopUp Card. I will then give feedback. Thank you for your help. 
By the way, the PopUp Card is really great.

**Comment by parkkralle** [comment]
The same problem exists on the pixel 8 pro.

**Comment by Clooos** [comment]
Then I guess that you have a pretty big config with many pop-ups? I will try to disable the blur when the pop-ups are closed, this should improve a lot that feature! 🤞

**Comment by parkkralle** [comment]
Unfortunately no, we only configured one single popup window.

**Comment by Clooos** [comment]
Then it might be related to your theme? Do you have a custom one?

**Comment by parkkralle** [comment]
Yes we use a custom theme. I will try the standard theme.

**Comment by parkkralle** [comment]
Same problem with the standard theme :(

**Comment by Clooos** [comment]
Then maybe a conflict with an other custom component? 😅

**Comment by parkkralle** [comment]
I'll take a screen capture tomorrow.

**Comment by Clooos** [comment]
Thanks!

**Comment by Clooos** [comment]
Could you try this version with the manual installation method? 

[Bubble Card v1.6.0-beta.3 test.zip](https://github.com/Clooos/Bubble-Card/files/13756532/Bubble.Card.v1.6.0-beta.3.test.zip)

**Comment by parkkralle** [comment]
Still the same problem.

**Comment by parkkralle** [comment]
The optimized mode also does not work on Android devices. When I first load the page I see the pop-ups render.

**Comment by Clooos** [comment]
I really don't understand how it's possible 😅 I've tested this version on a clean HA installation in a VM and everything works perfectly on iOS, Android, Safari, Chrome, Firefox... Even my personal setup is a complex one and it runs smoothly.

There should be some kind of conflict with an other custom card.

Could you also check if you have any errors in your logs?

**Comment by Clooos** [comment]
Hi, could you try the last beta available on HACS?

**Comment by parkkralle** [comment]
Sorry for the late reply but I was very busy over the holidays.
Beta 3 (from HACS) has the same problems for me (jerky scrolling when blur is active and popup cards visible when the page is opened for the first time.
Here is a list of custom components:
<img width="252" alt="image" src="https://github.com/Clooos/Bubble-Card/assets/59173472/3ee134b6-fec9-434d-9a76-36c91bbffe92">
<img width="241" alt="image" src="https://github.com/Clooos/Bubble-Card/assets/59173472/1fd1cb04-c598-412d-8b2a-a3e54e31c503">
<img width="240" alt="image" src="https://github.com/Clooos/Bubble-Card/assets/59173472/352c815f-1de9-4143-8a34-fc0e06017ff5">

**Comment by parkkralle** [comment]
Same behavior with beta 4. The rendering is also visible in the edge browser on win 11. It looks good on iOS. Is there a problem with the chrome engine?

**Comment by Clooos** [comment]
I really don't know, I'm totally unable to replicate this issue. Without the ability to test your setup, I'm clueless right now 😅

**Comment by Clooos** [comment]
Can you try with the latest beta?

https://github.com/Clooos/Bubble-Card/releases/tag/v1.6.0-beta.5

**Comment by parkkralle** [comment]
Hello, I have now carried out a few tests with the latest version. Unfortunately with little success. I've come to terms with the jerky scrolling and can live with the work around (deactivate blur).
The bigger problem now is rendering. I was able to speed up rendering significantly by uninstalling the layout card. But I still have to wait a few seconds for the page to finish rendering. Here's a video.

PS: You may be able to verify the problem with the layout card.

https://github.com/Clooos/Bubble-Card/assets/59173472/68d461db-ceac-4dc3-ab72-61e1354a8e19

**Comment by Clooos** [comment]
Thanks for the detailed feedback, can you tell me on what Home Assistant version you are? Because this issue seems to be fixed in the latest versions. And if it's already the case, have you tried the optimized mode for the pop-ups?

**Comment by parkkralle** [comment]
I am on the latest version.
![image](https://github.com/Clooos/Bubble-Card/assets/59173472/916792b0-0471-4973-9b4a-5c40b582b50f)

I tried both, regular mode and optimized mode.

**Comment by Clooos** [comment]
Then I guess that there is some kind of conflict somewhere. The best way would be to create a new blank view and see what is causing this.

**Comment by parkkralle** [comment]
I have now created a completely new dashboard with a new view. Nevertheless, I still have the rendering problem. I used very simple cards.

![image](https://github.com/Clooos/Bubble-Card/assets/59173472/a6c0f2f7-7346-418f-a51f-42d3e809d682)

**Comment by Clooos** [comment]
Hi, could you tell me if it's fixed in the new v1.7.1? I've changed something related to your issue.

https://github.com/Clooos/Bubble-Card/releases/tag/v1.7.1

**Comment by parkkralle** [comment]
The problem is solved. Thank you for that. I still have a small suggestion for improvement: it would be nice to have an option to always open the pop-up at the top. If I open a pop-up and scroll down, then close it and then reopen it, the pop-up is still at the bottom position.

**Comment by Clooos** [comment]
I've added this to my list, nice suggestion. I'm closing this since the original issue was fixed.
